arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -1154,7 +1154,7 @@ ::frame_image( )
 
       AVFrame picture;
       av_image_fill_arrays(picture.data, picture.linesize,
-                           (uint8_t*)d->current_image_memory->data(),
+                           static_cast<uint8_t *>(d->current_image_memory->data() ),
```

Comment:
pedantic: whitespace here is unbalanced; especially odd since it differs from the change a few lines below

Reply:
Done